OpenTrons/opentrons-api
# fix(components, protocol-designer): hug hotkey shortcut tag width

# Overview

Add optional shrinkToContent prop to Tag component

Closes [RQA-3667](https://opentrons.atlassian.net/browse/RQA-3667)

## Changelog

## Test Plan and Hands on Testing

- create or import a protocol and navigate to edit > timeline
- verify that hotkey shortcuts at bottom left of screen matches new designs (copy + container hug)
<img width="285" alt="Screenshot 2024-11-21 at 2 31 51 PM" src="https://github.com/user-attachments/assets/c0684687-8242-45a7-9cc2-a29d24118bd2">

## Review requests

- see test plan
- 
## Risk assessment

low

[RQA-3667]: https://opentrons.atlassian.net/browse/RQA-3667?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
